src/common/lib/DevTools.browser.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {createDevTools} from 'redux-devtools';
```

Comment:
Any other ordering other then simple sort lines is spending time with almost irrelevant issue I think.

Sorting JSX is easy, just write one property to one line. You want it anyway to maintain 80 line max length.